validation/validate_dc2_run2.1i_object_table.ipynb

```diff
@@ -280,19 +431,20 @@
     "    plt.ylabel('Dec [deg]')\n",
     "    plt.colorbar(shrink=0.5, label='objects / bin')\n",
     "\n",
-    "    # This region isn't quite a polygon.  The sides should be curved.\n",
-    "    wfd_region = Polygon(dc2_run2x_wfd, color='red', fill=False)\n",
-    "    ddf_region = Polygon(dc2_run2x_ddf, color='orange', fill=False)\n",
-    "    ax.add_patch(wfd_region)\n",
-    "    ax.add_patch(ddf_region)\n",
-    "    \n",
-    "    max_delta_ra = dc2_run2x_wfd[3][0] - dc2_run2x_wfd[2][0]\n",
-    "    delta_dec = dc2_run2x_wfd[1][1] - dc2_run2x_wfd[3][1]\n",
-    "    grow_buffer = 0.05\n",
-    "    ax.set_xlim(dc2_run2x_wfd[3][0] + max_delta_ra * grow_buffer,\n",
-    "                dc2_run2x_wfd[2][0] - max_delta_ra * grow_buffer)\n",
-    "    ax.set_ylim(dc2_run2x_wfd[3][1] - delta_dec * grow_buffer,\n",
-    "                dc2_run2x_wfd[1][1] + delta_dec * grow_buffer)"
+    "    if show_dc2_region:\n",
+    "        # This region isn't quite a polygon.  The sides should be curved.\n",
+    "        wfd_region = Polygon(dc2_run2x_wfd, color='red', fill=False)\n",
+    "        ddf_region = Polygon(dc2_run2x_ddf, color='orange', fill=False)\n",
+    "        ax.add_patch(wfd_region)\n",
+    "        ax.add_patch(ddf_region)\n",
+    "\n",
+    "        max_delta_ra = dc2_run2x_wfd[3][0] - dc2_run2x_wfd[2][0]\n",
+    "        delta_dec = dc2_run2x_wfd[1][1] - dc2_run2x_wfd[3][1]\n",
+    "        grow_buffer = 0.05\n",
+    "        ax.set_xlim(dc2_run2x_wfd[3][0] + max_delta_ra * grow_buffer,\n",
+    "                    dc2_run2x_wfd[2][0] - max_delta_ra * grow_buffer)\n",
+    "        ax.set_ylim(dc2_run2x_wfd[3][1] - delta_dec * grow_buffer,\n",
+    "                    dc2_run2x_wfd[1][1] + delta_dec * grow_buffer)"
```

Comment:
The text says: "We're missing a bit of the South-West (lower-left, "North Up, East Left") corner." The gap that I see is in the lower right (but maybe I'm not looking at the right gap?

Also, you aren't a fan of matplotlib's [`invert_xaxis`](https://matplotlib.org/3.1.0/api/_as_gen/matplotlib.axes.Axes.invert_xaxis.html)?

Reply:
For the `inside_trapezoid` function you might consider making a path and then using matplotlib [Path.contains_points](https://matplotlib.org/3.1.1/api/path_api.html#matplotlib.path.Path.contains_points). It won't be quite as fast, but more flexible in shape. I guess this is a question of what you want to teach.

Reply:
Agreed that in my overall practice I should use `invert_[xy]axis` more often.  In this particular Notebook I'm specifically controlling the limits for each of the `set_[xy]lim` calls.  E.g., in the above, I'm explicitly adding a buffer around the x limits.

There's one `plt.ylim(...)` that's used to flip.  I'll consider changing it.